versioned_docs/version-8.2/components/console/console-troubleshooting/feedback-and-support.md

```diff
@@ -3,8 +3,8 @@ id: feedback-and-support
 title: Feedback and support
```

Comment:
This page should be removed entirely. let's route all contact (problems, questions, suggestions, etc.) through the Contact page to make it easier to maintain.